tests/pytests/unit/modules/test_aix_status.py

```diff
@@ -0,0 +1,175 @@
+import logging
+import sys
+
+import salt.modules.status as status
+from tests.support.mock import MagicMock, patch
+
+try:
```

Comment:
Why would pytest not be available here?  And what happens when it's not?

Reply:
Removed try/except around import pytest